### DISCUSSION TITLE
ProCard 组件为什么要阻止冒泡呢? Why does the ProCard component use the stopPropagation method?

### DISCUSSION THREAD

**Discussion by itmier** [ant-design/pro-components#7566] [Q&A]
![image](https://github.com/ant-design/pro-components/assets/59595216/3a8ab283-7845-4d95-b0a1-abb63c66a43e)
我在写一个类文件夹组件, 需要在右侧图标处自定义按钮, 写这个组件的过程(该组件放置在ProCard)中, 对body的click事件进行了监听, 发现被捕获了, 不起作用, 简单看了下ProCard的源码, 发现在wrapDiv中就stopPropagation, 不理解为什么这么做, 有没有大佬来解答下?

**Comment by wzzz711** [comment]
Saski?
